### PR TITLE
DOC-811: Added PowerPaste release notes and documented gdocs support

### PR DESCRIPTION
**Pull request by lnewson**
Related Ticket: DOC-811

Description of Changes:
* Added new `powerpaste_googledocs_import` setting details
* Added PowerPaste 5.5.0 release notes
* Fixed some incorrect `setting` references (should be `option` in the docs)

**Note:** Known issues will be done in a separate PR.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/misc/support-powerpaste.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+PowerPaste (Copy and Paste) is supported on all the browsers supported by {{site.productname}} Enterprise. There is some variances of functionality for different browsers.
 
 |                       | HTML Cleaning | Improved HTML Cleaning | Image Import  |
 |-----------------------| :-----------: | :--------------------: | :-----------: |
 | Internet Explorer 11  | {{site.tick}} | {{site.cross}}         | {{site.cross}}|
 | Microsoft Edge Latest | {{site.tick}} | {{site.tick}}          | {{site.tick}} |
 | Chrome Latest         | {{site.tick}} | {{site.tick}}          | {{site.tick}} |
 | Firefox Latest        | {{site.tick}} | {{site.tick}}          | {{site.tick}} |
-| Safari Latest         | {{site.tick}} | {{site.tick}}          | {{site.cross}}|
+| Safari Latest         | {{site.tick}} | {{site.tick}}          | {{site.tick}} |
```
Comment (lnewson) [inline _includes/misc/support-powerpaste.md:18]: @metricjs as far as I know this has been working for quite a while now, so I've changed it but just wanted to confirm?

Comment (metricjs) [reply _includes/misc/support-powerpaste.md:18]: Unless "Image Import" means something different to what I'm thinking of, this works, yes.

File: release-notes/release-notes58.md [hunk 2]
```diff
@@ -101,6 +101,29 @@ The {{site.productname}} 5.8 release includes an accompanying release of the **E
...
 
+### PowerPaste 5.5.0
+
+The {{site.productname}} 5.8 release includes an accompanying release of the **PowerPaste** premium plugin.
+
+This release adds support for copying and pasting content from Google Docs, making it possible to copy and retain styles from Google Docs content. By default, a dialog will now appear asking users if the content should be cleaned or merged when pasting Google Docs content.
+
+**PowerPaste** 5.5.0 provides the following new features:
+
+- Added Google Docs paste support which can be controlled using the new [`powerpaste_googledocs_import`]({{site.baseurl}}/plugins/premium/powerpaste/#powerpaste_googledocs_import) option.
```
Comment (tylerkelly13) [inline release-notes/release-notes58.md:112]:
```suggestion
- Added Google Docs paste support which can be controlled using the new [`powerpaste_googledocs_import` option]({{site.baseurl}}/plugins/premium/powerpaste/#powerpaste_googledocs_import).
```

File: release-notes/release-notes58.md [hunk 3]
```diff
@@ -101,6 +101,29 @@ The {{site.productname}} 5.8 release includes an accompanying release of the **E
...
+
+**PowerPaste** 5.5.0 provides the following improvements:
+
+- The editor will now show a "loading" screen while paste events are processed.
+
+**PowerPaste** 5.5.0 provides the following bug fixes:
+
+- Fixed an issue where automatic linking didn't work with URLs containing commas in the path.
```
Comment (tylerkelly13) [inline release-notes/release-notes58.md:120]:
```suggestion
- Fixed an issue where automatic linking didn't work for URLs containing commas.
```

File: _includes/misc/support-powerpaste.md [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
 
 #### Improved HTML Cleaning
 
-On browsers that support HTML5 APIs, PowerPaste is able to use improved HTML cleaning techniques.  Improved HTML cleaning uses the HTML5 clipboard APIs to access the clipboard directly. The original document structure and formatting (when importing formatting) is more likely to be preserved. Where available this approach gives the highest fidelity copy and paste from Microsoft Word to HTML.
+On browsers that support HTML5 APIs, PowerPaste is able to use improved HTML cleaning techniques.  Improved HTML cleaning uses the HTML5 clipboard APIs to access the clipboard directly. The original document structure and formatting (when importing formatting) is more likely to be preserved. Where available this approach gives the highest fidelity copy and paste from supported applications to HTML.
```
Comment (tylerkelly13) [inline _includes/misc/support-powerpaste.md:26]:
```suggestion
On browsers that support HTML5 APIs, PowerPaste is able to use improved HTML cleaning techniques. Improved HTML cleaning uses the HTML5 clipboard APIs to access the clipboard directly. The original document structure and formatting (when importing formatting) is more likely to be preserved. Where available this approach gives the highest fidelity copy and paste from supported applications to HTML.
```

File: plugins/premium/powerpaste.md [hunk 5]
```diff
@@ -105,16 +107,34 @@ The supported values are:
...
+
+**Type:** `String`
+
+**Default value:** `prompt`
+
+The supported values are:
+
+* `clean` - Preserve the structure of the content such as headings, tables and lists but remove inline styles and classes. This results in simple content that uses the site's CSS stylesheet while retaining the semantic structure from the original document.
+* `merge` - Preserve the inline formatting and structure of the original document. Invalid and proprietary styles, tags and attributes are still removed ensuring that the HTML is valid while more closely matching the original document formatting.
+* `prompt` - Prompt the user to choose between the clean and merge options after attempting to paste Microsoft Word content.
```
[the remark this answers is not in the document]
Comment (lnewson) [reply plugins/premium/powerpaste.md:124]: :facepalm: thanks for the catch!

File: release-notes/release-notes58.md (hunk 3 above)
Comment (lnewson) [reply release-notes/release-notes58.md:120]: This is the same as what's distributed in the actual changelog. Are you sure you want it changed?

File: _includes/misc/support-powerpaste.md [hunk 6]
```diff
@@ -1,30 +1,30 @@
...
-{{site.companynameformal}} tests and supports using the PowerPaste plugin for copying content from versions of Microsoft Word and Microsoft Excel:
+{{site.companynameformal}} tests and supports using the PowerPaste plugin for copying content from Google Docs and versions of Microsoft Word and Microsoft Excel:
 
 * Covered by Microsoft Mainstream Support based on the [Microsoft Fixed Lifecycle Policy](https://support.microsoft.com/en-us/help/14085/fixed-lifecycle-policy).
 * Provided by Microsoft Office 365.
 
 ### PowerPaste Feature and Browser Support
 
-PowerPaste (Word copy and paste) is supported on all the browsers supported by {{site.productname}} Enterprise. There is some variances of functionality for different browsers.
+PowerPaste (Copy and Paste) is supported on all the browsers supported by {{site.productname}} Enterprise. There is some variances of functionality for different browsers.
```
Comment (ashfordneil) [inline _includes/misc/support-powerpaste.md:10]:
```suggestion
PowerPaste (Copy and Paste) is supported on all the browsers supported by {{site.productname}} Enterprise. There are some variances of functionality for different browsers.
```

**Review by ashfordneil — APPROVED** [review]

File: _includes/misc/support-powerpaste.md [hunk 7]
```diff
@@ -1,30 +1,30 @@
-## PowerPaste support (Word Copy and Paste)
+## PowerPaste support (Copy and Paste)
 
-{{site.companynameformal}} tests and supports using the PowerPaste plugin for copying content from versions of Microsoft Word and Microsoft Excel:
+{{site.companynameformal}} tests and supports using the PowerPaste plugin for copying content from Google Docs and versions of Microsoft Word and Microsoft Excel:
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/misc/support-powerpaste.md:3]: Update: We are currently determining the support scope. PR will be updated ASAP

File: release-notes/release-notes58.md (hunk 3 above)
Comment (tylerkelly13) [reply release-notes/release-notes58.md:120]: dang. ok ignore me

File: _includes/misc/support-powerpaste.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/misc/support-powerpaste.md]: Updated in [`b7d20ee` (#1955)](https://github.com/tinymce/tinymce-docs/pull/1955/commits/b7d20ee19e32097bc446e1561b978c7ecef78f3a)

File: plugins/premium/powerpaste.md [hunk 8]
```diff
@@ -105,16 +107,34 @@ The supported values are:
...
+The supported values are:
+
+* `clean` - Preserve the structure of the content such as headings, tables and lists but remove inline styles and classes. This results in simple content that uses the site's CSS stylesheet while retaining the semantic structure from the original document.
+* `merge` - Preserve the inline formatting and structure of the original document. Invalid and proprietary styles, tags and attributes are still removed ensuring that the HTML is valid while more closely matching the original document formatting.
+* `prompt` - Prompt the user to choose between the clean and merge options after attempting to paste Google Docs content.
+
 ### powerpaste_html_import
 
-This setting controls how content pasted from sources other than Microsoft Word is filtered. Note that this includes content copied from {{site.productname}} itself.
+This option controls how content pasted from sources other than Microsoft Word and Google Docs is filtered. Note that this includes content copied from {{site.productname}} itself.
```
Comment (tylerkelly13) [inline plugins/premium/powerpaste.md:128]:
```suggestion
This option controls how content pasted from sources other than Microsoft Word and Google Docs are filtered. Note that this includes content copied from {{site.productname}} itself.
```

File: release-notes/release-notes58.md [hunk 9]
```diff
@@ -101,6 +101,32 @@ The {{site.productname}} 5.8 release includes an accompanying release of the **E
 
 For information on the Export plugin, see: [Export plugin]({{site.baseurl}}/plugins/premium/export/).
 
+### PowerPaste 5.5.0
+
+The {{site.productname}} 5.8 release includes an accompanying release of the **PowerPaste** premium plugin.
+
+This release adds support for copying and pasting content from Google Docs, making it possible to copy and retain styles from Google Docs content. By default, a dialog will now appear asking users if the content should be cleaned or merged when pasting Google Docs content.
```
Comment (tylerkelly13) [inline release-notes/release-notes58.md:108]:
```suggestion
This release adds support for copying and pasting content from Google Docs, making it possible to copy content and retain styles from Google Docs. By default, a dialog will appear asking users if the content should be cleaned or merged when pasting Google Docs content.
```
With `content` there, it reads like we're copying the styles, not the content and styles

**Review by tylerkelly13 — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**